Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_hole.h

```diff
@@ -42,8 +43,9 @@ namespace Polygon_mesh_processing {
   /*!
   \ingroup hole_filling_grp
   triangulates a hole in a polygon mesh.
```

Comment:
I would like to suggest some rewording.  And I would remove the "However.." sentence
as it is not about preconditions.

Depending on the choice of the underlying algorithm different preconditions apply.
When using the 2D constrained Delaunay triangulation, the border edges of the hole
must not intersect the surface.  Otherwise,  additionally, the boundary
of the hole must not contain any non-manifold vertex.

Reply:
Yes, this sounds better. I have fixed that.